source/features/reactions-avatars.css

```
@@ -18,32 +28,36 @@
 	font-size: 10px; /* Base sizer */
 	transition: margin-left 0.2s;
 }
-.reaction-summary-item a:first-of-type {
+
+.rgh-reactions .reaction-summary-item a:first-of-type {
 	margin-left: 0.5em;
 }
-.review-comment .reaction-summary-item a {
+
+.review-comment .rgh-reactions .reaction-summary-item a {
 	font-size: 9px;
 }
-.discussion-post .reaction-summary-item a {
+
+.discussion-post .rgh-reactions .reaction-summary-item a {
 	margin-top: -1px;
 }
 
 /* This image will start at height:0 and will expand once loaded, covering the gray placeholder */
-.reaction-summary-item img {
+.rgh-reactions .reaction-summary-item img {
```

This `img` doesn't exist outside the feature so this selector isn't needed
```suggestion
.reaction-summary-item img {
```

But perhaps it can still be kept for consistency.